### ISSUE TITLE
Dialog initial size

### ISSUE THREAD

**Issue by pelizza** [microsoft/azure-devops-extension-api#49]
When defining an external dialog content like this:
```
    {
      "id": "my-dialog",
      "type": "ms.vss-web.external-content",
      "description": "My dialog",
      "targets": [],
      "properties": {
        "uri": "myDialog.html"
      }
    },
```

...and opening a custom dialog like this:
```
SDK.getService(CommonServiceIds.HostPageLayoutService).then(
      dialogService => {
        return dialogService.openCustomDialog(SDK.getExtensionContext().id + ".my-dialog", {
          title: "My dialog title",
          onClose: result => console.log(result)
        });
      }
    );
```

...it is not possible to set the dialog size beforehand. Alternatively, the dialog size needs to be set when the dialog content is initialized:
```
// myDialog.js
SDK.init();
await SDK.ready();
SDK.resize(440, 400);
```

This makes the UI experience not great because the dialog is firstly rendered very small and only after that the resize happens. It was possible to set the dialog size in advance when using the old SDK though.

Is this a missing feature, a bug, or maybe it is just not documented?

Thank you.

**Comment by karelkral** [comment]
Hello @pelizza, I have a similar experience. But my experience is that SDK.Resize does not even change the dialog's width, only height. This issue reports this also: https://github.com/microsoft/azure-devops-extension-sdk/issues/18

**Comment by 51manjunath** [comment]
Hi @karelkral  - We're having a same problem. Have you found a solution to this problem?

**Comment by karelkral** [comment]
No, I have not. No answer from MS. In my opinion Azure DevOps is completely unsupported.
My solution was completely left the custom dialogs of this type. 

I am using CustomDialog component from [SDK](https://developer.microsoft.com/en-us/azure-devops/components/dialog#customdialog) and this is a good component. But cannot use it as a replacement for the case above.

**Comment by karelkral** [comment]
SDK dialog cannot be called from all contexts. Basically, you can call from another React view. You cannot call it an entry point of the extension.

**Comment by TechnoRahmon** [comment]
I have same issue, any update ?
